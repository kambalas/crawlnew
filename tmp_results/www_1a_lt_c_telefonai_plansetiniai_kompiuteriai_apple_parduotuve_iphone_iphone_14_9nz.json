[
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIConnectionError: 'Message' object has no attribute 'audio'\nTraceback (most recent call last):\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\main.py\", line 1022, in completion\n    model_response = ModelResponse()\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 1168, in __init__\n    choices = [Choices()]\n               ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 778, in __init__\n    self.message = Message()\n                   ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 636, in __init__\n    del self.audio\n        ^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\pydantic\\main.py\", line 1072, in __delattr__\n    object.__delattr__(self, item)\nAttributeError: 'Message' object has no attribute 'audio'\n"
  },
  {
    "index": 1,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "litellm.APIConnectionError: 'Message' object has no attribute 'audio'\nTraceback (most recent call last):\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\main.py\", line 1022, in completion\n    model_response = ModelResponse()\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 1168, in __init__\n    choices = [Choices()]\n               ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 778, in __init__\n    self.message = Message()\n                   ^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\litellm\\types\\utils.py\", line 636, in __init__\n    del self.audio\n        ^^^^^^^^^^\n  File \"C:\\Miniconda\\Lib\\site-packages\\pydantic\\main.py\", line 1072, in __delattr__\n    object.__delattr__(self, item)\nAttributeError: 'Message' object has no attribute 'audio'\n"
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro 1TB Deep Purple",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro 512GB Silver",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 512GB RED",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro Max 512GB Deep Purple",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 128GB Yellow",
    "price": "758,00",
    "error": false
  },
  {
    "index": 3,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]