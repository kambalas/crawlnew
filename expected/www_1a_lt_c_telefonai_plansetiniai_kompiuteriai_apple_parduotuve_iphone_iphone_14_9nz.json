[
  {
    "name": "iPhone 14",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 14 128GB (Product)RED",
    "price": "849",
    "error": false
  },
  {
    "name": "iPhone 14 128GB Blue",
    "price": "849",
    "error": false
  },
  {
    "name": "iPhone 14 128GB Midnight",
    "price": "849",
    "error": false
  },
  {
    "name": "iPhone 14 128GB Starlight",
    "price": "849",
    "error": false
  },
  {
    "name": "iPhone 14 256GB (Product)RED",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 14 256GB Blue",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 14 256GB Midnight",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 14 256GB Starlight",
    "price": "999",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 128GB Yellow",
    "price": "758,00 €",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 128 GB, raudona",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 128 GB, violetinė",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 256 GB, balta",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 128 GB, midnight",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 256GB Yellow",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 128 GB, aukso",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 512GB Yellow",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 256GB Yellow",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14, 128 GB, midnight",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro Max 512GB Space Black",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro 256GB Space Black",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 512GB Blue",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro Max 256GB Silver",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro 128GB Space Black",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro Max 128GB Gold",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 512GB Midnight",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 256GB Purple",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 128GB Blue",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 128GB Purple",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro 512GB Silver",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 512GB RED",
    "price": "",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 14 Pro Max 512GB Deep Purple",
    "price": "",
    "error": false
  }
]